### PR TITLE
update configuration files for #369

### PR DESCRIPTION
**Pull request by hglkrijger**
update configuration files for #369

### CODE REVIEW

File: config/alpine/waagent.conf [hunk 1]
```diff
@@ -69,3 +58,9 @@ OS.RootDeviceScsiTimeout=300
 
 # If "None", the system default version is used.
 OS.OpensslPath=None
+
+# Enable or disable goal state processing auto-update, default is enabled
+# AutoUpdate.Enabled=y
```
Comment (ahmetalpbalkan) [inline config/alpine/waagent.conf]: why are these commented? e.g. ubuntu/waagent.conf has

```

# Enable or disable self-update, default is enabled
AutoUpdate.Enabled=y
AutoUpdate.GAFamily=Prod

```

Comment (hglkrijger) [inline config/alpine/waagent.conf]: I changed them all to be commented out, based on the issues which were opened and resulting discussion.

**Comment by ahmetalpbalkan** [comment]
ah ok, LGTM

**Pull request merged**